ionic2/app/pages/wiki-index/wiki-index.page.ts

```diff
@@ -21,8 +21,13 @@ export class WikiIndexPage {
   constructor(private navCtrl: NavController, private apiservice: ApiService) { }
 
   ionViewDidEnter() {
```

Comment:
Do we have a use case where this query string might be pre-filled? Are we doing this when we jump back from the detail view?